includes/Services/FlowService.php

```diff
@@ -0,0 +1,191 @@
+<?php
```

Comment:
![Screenshot 2022-12-05 at 9 41 49 PM](https://user-images.githubusercontent.com/38878906/205685963-3e5fc404-36dc-48dc-adcb-aa29088f68c5.png)
I vaguely remember discussing this but mentioning again in case we did not, phpcs and phpcbf both are failing on this file which means these will also fail in our release pipelines.